.vscode/launch.json

```
@@ -4,6 +4,18 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+
+    {
+      "name": "(Windows) Launch",
+      "type": "cppvsdbg",
+      "request": "launch",
+      "program": "${workspaceFolder}/build_default_relwithdebinfo/bin/Orbit",
+      "args": [],
+      "stopAtEntry": false,
+      "cwd": "${workspaceFolder}/build_default_relwithdebinfo/bin/",
+      "environment": [],
+      "externalConsole": false
```

```suggestion
      "externalConsole": false,
      "preLaunchTask": "shell: build default relwithdebinfo"
```

This would always trigger a build before starting the debugging. This is how I have my configuration, but it is purely preference